### DISCUSSION TITLE
miscellaneous comments

### DISCUSSION THREAD

**Discussion by JisuJung928** [MDIL-SNU/SIMPLE-NN_v2#3] [General]
https://github.com/MDIL-SNU/SIMPLE-NN_v2/blob/14b4c14e6d77bfa1b03ddc6da358b8d3dc43fe98/test_sf/test_sf.py#L88

For readability, please define angular ACSF including 2**(1-zeta) as it is written (J. Chem. Phys. 134, 074106 (2011)).

https://github.com/MDIL-SNU/SIMPLE-NN_v2/blob/14b4c14e6d77bfa1b03ddc6da358b8d3dc43fe98/test_sf/test_sf.py#L18

How about replacing all functions of `math`  with `numpy` for `dtype` management and a faster matrix calculation someday?

https://github.com/MDIL-SNU/SIMPLE-NN_v2/blob/fde2d083c851fdf76bd101a7e244aca777065765/symmetry_function/__init__.py#L103

How about dealing with `res` variable in inline for avoiding overhead from calling the function in python?

**Comment by Nanco-L** [comment]
Overhead from calling the function is good point that we need to check.
In addition, I suggest to avoid writing code in `__init__.py`.
(I know that I wrote the code in `features/symmetry_function/__init__.py` for the current version of `SIMPLE-NN`.. But it is not recommended.)
